vars/retrievePreviousBuild.groovy

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+/** Library to call build.sh script (builds OpenSearch and OpenSearch Dashboards distribution)
```

Comment:
Fix the lib definition? I believe this lib only downloads the  previous builds?

Reply:
Updated.

Reply:
Hi @zelinh I still don't see the updated definition of the library. Was this missed?

Reply:
Seems like I updated at wrong place. Just made a new commit on this.